native/packages/react-native-bpk-component-button/src/styles.js

```diff
@@ -28,79 +28,52 @@ import {
   colorGreen500,
   colorPink500,
   colorPink600,
-  buttonHeight,
-  buttonPaddingVertical,
-  buttonPaddingHorizontal,
   buttonBorderWidth,
-  buttonLineHeightLarge,
-  lineHeightXs,
   borderRadiusPill,
   spacingSm,
+  spacingMd,
```

Comment:
If we are no longer using the button token values, should they in fact be removed?

Reply:
That would be a breaking change that I want to avoid just now.

Reply:
The idea of component specific tokens were well intentioned. But in practice they end up making the code complex and in reality nobody uses them.